src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const allCountriesInfo = document.querySelectorAll('.population');
+const totalElement = document.querySelector('.total-population');
+const averageElement = document.querySelector('.average-population');
+
+const populationValues = [...allCountriesInfo]
+  .map(item => parseInt(item.innerText.split(',').join('')));
+const totalAmount = populationValues.reduce((a, b) => a + b);
```

Comment:
```suggestion
const totalAmount = populationValues.reduce((a, b) => a + b, 0);
```

better use initial value in your `reduce`